ingestion_server/ingestion_server/es_mapping.py

```diff
@@ -51,109 +66,77 @@ def index_settings(table_name):
+            "category": {"type": "keyword"},
+            # Text-based fields
             "title": {
                 "type": "text",
+                "analyzer": "custom_english",
                 "similarity": "boolean",
                 "fields": {
                     "keyword": {"type": "keyword", "ignore_above": 256},
                     "raw": {"type": "text", "index": True},
                 },
-                "analyzer": "custom_english",
-            },
-            "foreign_landing_url": {
-                "fields": {"keyword": {"ignore_above": 256, "type": "keyword"}},
-                "type": "text",
             },
             "description": {
-                "fields": {
-                    "keyword": {"type": "keyword", "similarity": "boolean"},
-                    "raw": {"type": "text", "index": True},
-                },
                 "type": "text",
                 "analyzer": "custom_english",
+                "similarity": "boolean",
+                "fields": {"raw": {"type": "text", "index": True}},
             },
             "creator": {
                 "type": "text",
                 "fields": {"keyword": {"type": "keyword", "ignore_above": 256}},
             },
-            "url": {
-                "fields": {"keyword": {"type": "keyword", "ignore_above": 256}},
-                "type": "text",
-            },
-            "extension": {
-                "fields": {"keyword": {"ignore_above": 8, "type": "keyword"}},
-                "type": "text",
-            },
-            "license": {
-                "fields": {"keyword": {"ignore_above": 256, "type": "keyword"}},
-                "type": "text",
-            },
-            "license_version": {
-                "type": "text",
-                "fields": {"keyword": {"type": "keyword", "ignore_above": 256}},
-            },
-            "license_url": {
-                "fields": {"keyword": {"type": "keyword", "ignore_above": 256}},
-                "type": "text",
-            },
-            "provider": {
-                "type": "text",
-                "fields": {"keyword": {"type": "keyword", "ignore_above": 256}},
-            },
-            "source": {
-                "fields": {"keyword": {"ignore_above": 256, "type": "keyword"}},
-                "type": "text",
+            # Rank feature fields
+            "standardized_popularity": {"type": "rank_feature"},
+            "authority_boost": {"type": "rank_feature"},
+            "authority_penalty": {
+                "type": "rank_feature",
+                "positive_score_impact": False,
             },
-            "filetype": {"type": "keyword"},
-            "created_on": {"type": "date"},
+            "max_boost": {"type": "rank_feature"},
+            "min_boost": {"type": "rank_feature"},
+            # Nested fields
             "tags": {
                 "properties": {
                     "accuracy": {"type": "float"},
+                    # Text-based fields
```

Comment:
I think setting it to keyword-only would greatly reduce the size of the index. It would change the search by tags a little bit. I.e., it would probably not match if you are searching for `draw`, you won't get the media that has `drawing` tag, but do we really want it to match like that?

Reply:
This is an important point, we could convert it into keywords (and potentially use stems so that 'drawing' and 'draw' evaluate to the same keyword). I'll let others chime in on this discussion as well.

Reply:
Could we maybe see how this set of changes goes, then consider moving tags to keywords as a follow-up? I agree that keywords + stemming could be useful (if that's a distinction we can make), but it would be a significant change to the way our search currently operates which leaves me a little hesitant.

Reply:
True, we can keep this possibility open as a future optimisation.

Reply:
When discussing the collection views we explicitly said that tag stemming was desirable. The tag `dogs` should match `dog`, we used as an example. Any other approach doesn't make much sense and would confound users (and ourselves, I have to think!).